ts/project.test.ts

```diff
@@ -50,3 +56,157 @@ const _testTypeCheckingOfAddExecutableTemplate = (project: Project) => {
   // @ts-expect-error - shell should be the actual executable now, not the helper
   p.shell``;
 };
+
+describe("Project.add", () => {
+  it("allows to add fields with .add", () => {
```

Comment:
```suggestion
  it("allows adding fields with .add", () => {
```
here and in a few places below

Reply:
Done in a515a1ea.